test_collections/matter/sdk_tests/support/sdk_container.py

```diff
@@ -176,6 +179,10 @@ async def start(self) -> None:
             self.image_tag, self.run_parameters
         )
 
+        BaseManager.register("TestRunnerHooks", SDKPythonTestRunnerHooks)
+        self.manager = BaseManager(address=("0.0.0.0", 50000), authkey=b"abc")
+        self.manager.start()
```

Comment:
Maybe it would be good to mock these `register` and `start` method calls on any unit tests that execute this method

Reply:
Looks good!